### PR TITLE
chore(functions/tips-lazy-globals): Update sample signature using functions framework

### PR DESCRIPTION
**Pull request by tyampad**
Update [Lazy globals sample](http://cloud/functions/docs/samples/functions-tips-lazy-globals) to use functions framework.

Tested by:

Running local tests
Deploying and invoking using curl

### CODE REVIEW

**Comment by ace-n** [comment]
@tyampad thanks for the PR! Can you fix the lint error?

> ./main.py:31:1: E402 module level import not at top of file
import functions_framework
^
nox > Command flake8 --show-source --builtin=gettext --max-complexity=20 --import-order-style=google '--exclude=.nox,.cache,env,lib,generated_pb2,*_pb2.py,*_pb2_grpc.py' --ignore=ANN101,E121,E123,E126,E203,E226,E24,E266,E501,E704,W503,W504,I202 --max-line-length=88 --application-import-names main_test,noxfile,.nox,main . failed with exit code 1
nox > Session lint failed.
PWD: /workspace/functions/tips-lazy-globals

File: functions/tips-lazy-globals/main.py [hunk 1]
```diff
@@ -28,13 +28,17 @@ def function_specific_computation():
 
 # [START functions_tips_lazy_globals]
 # [START cloudrun_tips_global_lazy]
+import functions_framework
```
Comment (ace-n) [inline functions/tips-lazy-globals/main.py:31]: To fix the lint error:
1) Move this import statement to the top of the file (e.g. line 15)
2) Break the sample up into multiple region tags, a la:
```
// [START tag]
<imports go here>
// [END tag]

...

// [START tag]
<code goes here>
// [END tag]
```

**Review by ace-n — CHANGES_REQUESTED** [review]

File: functions/tips-lazy-globals/main.py [hunk 2]
```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# [START functions_tips_lazy_globals]
+# [START cloudrun_tips_global_lazy]
+import functions_framework
```
Comment (ace-n) [inline functions/tips-lazy-globals/main.py:17]: Nit: add newline after import statement _within the region-tagged block_
```suggestion
import functions_framework

```

File: functions/tips-lazy-globals/main.py [hunk 3]
```diff
@@ -12,6 +12,11 @@
...
+# [END cloudrun_tips_global_lazy]
+# [END functions_tips_lazy_globals]
 
 from functools import reduce  # noqa I100
```
Comment (ace-n) [inline functions/tips-lazy-globals/main.py:21]: Can we delete this import?

**Review by ace-n — CHANGES_REQUESTED** [review]

**Review by ace-n — APPROVED** [review]

**Review by nicain — APPROVED** [review]

**Pull request merged**